v2/types.go

```diff
@@ -1364,6 +1364,13 @@ type CandleSnapshot struct {
 	Snapshot []*Candle
 }
 
+type CandleHistoryParams struct {
+	StartMTS int64
+	EndMTS int64
+	Limit int
+	Sort int
```

Comment:
As a user of the Bitfinex go API, I would want to minimize referencing API docs to make calls. I think it would be helpful if `Sort` had a typedef, from docs:

> if = 1 it sorts results returned with old > new

feel free to define 1 & any other const you'd like for an Ascending & Descending const, provided the consts work with the API in practice